tests/kola/misc/test-status.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash     
+
+# Simple sanity checks for systemd unit status updates.
+
+set -xeuo pipefail
+
+. ${KOLA_EXT_DATA}/libtest.sh
+
+cd $(mktemp -d)
+
+systemctl stop zincati.service
+echo "updates.enabled = false" > /etc/zincati/config.d/disable-in-test.toml
+systemctl start zincati.service
+systemctl status zincati.service | grep 'Status:' > zincati_disabled_status.txt
```

Comment:
There's also e.g. `systemctl show -p StatusText zincati.service` which is more precise.